static/js/tools/choropleth/choropleth.tsx

```diff
@@ -356,4 +373,38 @@ function determineColorPalette(dict, pc: boolean, popMap: []): number[] {
   return [lowerValue, medianValue, upperValue];
 }
 
+/**
+ * Returns a nicely formatted hover value for the value of statistical variables
+ * for geographies.
+ * @param geoValue statistical variable value to display.
+ * @param isPerCapita whether the value is a per capita indicator.
+ */
+function formatGeoValue(geoValue, isPerCapita) {
+  // Per capita values may be difficult to read so add a per count field.
+  // E.g. 0.0052 or 5.2 per 1000.
+  if (isPerCapita || geoValue === "No Value") {
+    // Find a multiplier such that the value is greater than 1.
+    if (geoValue < 1) {
+      let multiplier = 1;
+      let dispValue = geoValue;
+      while (dispValue && dispValue < 1) {
+        dispValue *= 10;
+        multiplier *= 10;
+      }
+      return (
+        geoValue.toFixed(6) +
+        " or " +
+        dispValue.toLocaleString() +
+        " per " +
+        multiplier.toLocaleString() +
+        " people."
```

Comment:
This can also be:

`{geoValue.toFixed(6)} or {dispValue.toLocaleString()} per {multiplier.toLocaleString()} people`

Reply:
Updated to template